testing/sdk_marathon.py

```diff
@@ -41,6 +41,14 @@ def destroy_app(app_name):
     shakedown.delete_app_wait(app_name)
 
 
+def restart_app(app_name):
+    sdk_utils.out("Restarting {}...".format(app_name))
```

Comment:
(for what it's worth, I don't think one should enforce anything that cannot be checked automatically)